cyral/resource_cyral_repository_test.go

```diff
@@ -196,8 +220,12 @@ func repoCheckFuctions(repo RepoInfo, resName string) resource.TestCheckFunc {
 				repo.MongoDBSettings.ReplicaSetName),
 
 			resource.TestCheckResourceAttr(resourceFullName,
-				"connection_draining.0.server_type",
+				"mongodb_settings.0.server_type",
```

Comment:
@hbaackmann I think this is a bug from the current tip of the `feature/multiplexing` branch, but it looks like we did not run any tests with mongodb_settings, so we never caught it.